bofire/data_models/enum.py

```diff
@@ -43,3 +43,10 @@ class RegressionMetricsEnum(Enum):
     PEARSON = "PEARSON"
     SPEARMAN = "SPEARMAN"
     FISHER = "FISHER"
+    PEARSON_UQ = "PEARSON_UQ"
```

Comment:
Create a new enum named UQRegressionMetricsEnum and move the UQ Metrics into this,